diffpy/snmf/factorizers.py

```diff
@@ -3,9 +3,10 @@
 
 
 def lsqnonneg(stretched_component_matrix, target_signal):
-    """
-    Solves ``argmin_x || Ax - b ||_2`` for ``x>=0``. Finds the weights of component signals given unfactorized signal
-    data and stretched components.
+    """Finds the weights of component signals given unfactorized signal data and stretched components.
```

Comment:
please check this.  Is it getting the weights or the stretch-factors?  Let's be double-sure.  If you are not sure, check with Ran.  We don't want to merge if it is not right....lots of hassle in the future.....

Reply:
I have had a meeting with Ran. He said that they get the stretching factors.

Reply:
Since it is finding the stretching factors don't we need to change the docstring to say this? `Finds the stretching factor of each component signal given an input signal and the components.` Something like that.  If it is over 80 chars shorten it and put more detail below

Reply:
Sorry. I had a typo in a previous comment, it does find the weights of stretched components. It does not find stretching factors.